### ISSUE TITLE
How can I deploy this repo to Azure?

### ISSUE THREAD

**Issue by saidi59** [mckaywrigley/chatbot-ui#1866]
How can I deploy this repo to Azure?

**Comment by shortyfactory** [comment]
did you find any solution or still need help ?

**Comment by saidi59** [comment]
Regrettably, I have been unable to identify a solution to the problem.
I would appreciate assistance in resolving this matter.

**Comment by stephanexenos** [comment]
Hey, do you still need to do this ? 
I can provide some assistance

**Comment by saidi59** [comment]
Yes, I need help, I can’t deploy it azure web app also azure function app.
Please help me
Thanks

**Comment by madhumitha-santhanakrishnan** [comment]
Hey, could someone help me with this? I'm stuck when trying to deploy this to azure.
